### PR TITLE
Tasmota introduction

### PR DESCRIPTION
**Pull request by s-hadinger**
### Description of the feature or fix

Introduction of Tasmota+Berry in the LVGL documentation with a quick start on M5Stack devices in less than 10 minutes

### Checkpoints
- [ ] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [x] Update the documentation

### CODE REVIEW

File: docs/get-started/index.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ There are several ways to get your feet wet with LVGL. Here is one recommended o
 3. Read the [Quick overview](https://docs.lvgl.io/latest/en/html/get-started/quick-overview.html) page of the documentation (15 minutes)
 4. Set up a [Simulator](https://docs.lvgl.io/latest/en/html/get-started/pc-simulator.html) (10 minutes)
 5. Try out some [Examples](https://github.com/lvgl/lv_examples/)
+6. Try LVGL on ESP32 M5Stack device with [Tasmota and Berry](https://tasmota.github.io/docs/LVGL_in_10_minutes/) (10 minutes)
 6. Port LVGL to a board. See the [Porting](https://docs.lvgl.io/latest/en/html/porting/index.html) guide or check the ready to use [Projects](https://github.com/lvgl?q=lv_port_&type=&language=)
```
Comment (mysterywolf) [inline docs/get-started/index.md]: I think it should be 7 now, and 8, 9 ...

Comment (kisvegabor) [reply docs/get-started/index.md]: As we do not mention another platform here either, it wouldn't be fair to mention only Tasmota. 

What about 
```suggestion
6. Check out the [Getting started](https://docs.lvgl.io/master/get-started/index.html) guides for platform-specific tutorials. 
```

**Review by kisvegabor — COMMENTED** [review]
Looks good, thank you very much! Only a minor comment above.

File: docs/get-started/index.md (hunk 1 above)
Comment (s-hadinger) [reply docs/get-started/index.md]: Thanks, that's fair indeed. However the link at 6. is now pointing to the same page, isnt't it? Or am I missing somehing?

File: docs/get-started/index.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ There are several ways to get your feet wet with LVGL. Here is one recommended o
 3. Read the [Quick overview](https://docs.lvgl.io/latest/en/html/get-started/quick-overview.html) page of the documentation (15 minutes)
 4. Set up a [Simulator](https://docs.lvgl.io/latest/en/html/get-started/pc-simulator.html) (10 minutes)
 5. Try out some [Examples](https://github.com/lvgl/lv_examples/)
-6. Port LVGL to a board. See the [Porting](https://docs.lvgl.io/latest/en/html/porting/index.html) guide or check the ready to use [Projects](https://github.com/lvgl?q=lv_port_&type=&language=)
+6. Try LVGL on ESP32 M5Stack device with [Tasmota and Berry](https://tasmota.github.io/docs/LVGL_in_10_minutes/) (10 minutes)
+6. Check out the [Getting started](https://docs.lvgl.io/master/get-started/index.html) guides for platform-specific tutorials.
```
Comment (kisvegabor) [inline docs/get-started/index.md]:
```suggestion
6. Check out the [Getting started](https://docs.lvgl.io/master/get-started/index.html) guides for platform-specific tutorials. 
7. Port LVGL to a board. See the [Porting](https://docs.lvgl.io/latest/en/html/porting/index.html) guide or check the ready to use [Projects](https://github.com/lvgl?q=lv_port_&type=&language=)
```

Sorry, I have added the suggestion to the wrong point...

File: docs/get-started/index.md [hunk 3]
```diff
@@ -10,7 +10,8 @@ There are several ways to get your feet wet with LVGL. Here is one recommended o
 3. Read the [Quick overview](https://docs.lvgl.io/latest/en/html/get-started/quick-overview.html) page of the documentation (15 minutes)
 4. Set up a [Simulator](https://docs.lvgl.io/latest/en/html/get-started/pc-simulator.html) (10 minutes)
 5. Try out some [Examples](https://github.com/lvgl/lv_examples/)
-6. Port LVGL to a board. See the [Porting](https://docs.lvgl.io/latest/en/html/porting/index.html) guide or check the ready to use [Projects](https://github.com/lvgl?q=lv_port_&type=&language=)
+6. Check out the [Getting started](https://docs.lvgl.io/master/get-started/index.html) guides for platform-specific tutorials. 
+7. Port LVGL to a board. See the [Porting](https://docs.lvgl.io/latest/en/html/porting/index.html) guide or check the ready to use [Projects](https://github.com/lvgl?q=lv_port_&type=&language=)
 7. Read the [Overview](https://docs.lvgl.io/latest/en/html/overview/index.html) page to get a better understanding of the library. (2-3 hours)
 8. Check the documentation of the [Widgets](https://docs.lvgl.io/latest/en/html/widgets/index.html) to see their features and usage
 9. If you have questions got to the [Forum](http://forum.lvgl.io/)
```
Comment (kisvegabor) [inline docs/get-started/index.md]:
```suggestion
8. Read the [Overview](https://docs.lvgl.io/latest/en/html/overview/index.html) page to get a better understanding of the library. (2-3 hours)
9. Check the documentation of the [Widgets](https://docs.lvgl.io/latest/en/html/widgets/index.html) to see their features and usage
10. If you have questions got to the [Forum](http://forum.lvgl.io/)
```

File: docs/get-started/index.md [hunk 4]
```diff
@@ -10,10 +10,11 @@ There are several ways to get your feet wet with LVGL. Here is one recommended o
...
-7. Read the [Overview](https://docs.lvgl.io/latest/en/html/overview/index.html) page to get a better understanding of the library. (2-3 hours)
-8. Check the documentation of the [Widgets](https://docs.lvgl.io/latest/en/html/widgets/index.html) to see their features and usage
-9. If you have questions got to the [Forum](http://forum.lvgl.io/)
+6. Check out the [Getting started](https://docs.lvgl.io/master/get-started/index.html) guides for platform-specific tutorials. 
+7. Port LVGL to a board. See the [Porting](https://docs.lvgl.io/latest/en/html/porting/index.html) guide or check the ready to use [Projects](https://github.com/lvgl?q=lv_port_&type=&language=)
+8. Read the [Overview](https://docs.lvgl.io/latest/en/html/overview/index.html) page to get a better understanding of the library. (2-3 hours)
+9. Check the documentation of the [Widgets](https://docs.lvgl.io/latest/en/html/widgets/index.html) to see their features and usage
+10. If you have questions got to the [Forum](http://forum.lvgl.io/)
 10. Read the [Contributing](https://docs.lvgl.io/latest/en/html/contributing/index.html) guide to see how you can help to improve LVGL (15 minutes)
```
Comment (kisvegabor) [inline docs/get-started/index.md]:
```suggestion
11. Read the [Contributing](https://docs.lvgl.io/latest/en/html/contributing/index.html) guide to see how you can help to improve LVGL (15 minutes) 
```

**Comment by kisvegabor** [comment]
Sorry, I added the suggestion to the wrong point. Now it should be fine. I also corrected the numbers of the enumeration. 

Do you agree it's ok this way?

**Comment by s-hadinger** [comment]
Yes, still I'm confused that the link for item 6. points to itself. Wouldn't it be better to mention the different platform specific tutorials here?

**Comment by kisvegabor** [comment]
True, I've added a subsection for [platform specific links](https://github.com/lvgl/lvgl/commit/e7f1356d0dc5ad16a8ebe0148f3eef99f1b8e031) and pointed the link in step 6 to that section.

It's still not perfect but at least makes a little more sense.

**Comment by s-hadinger** [comment]
Sounds good to me!

**Comment by kisvegabor** [comment]
Great, merging! Than you very much!

**Comment by kisvegabor** [comment]
I made it a little bit simpler:
image


https://docs.lvgl.io/master/get-started/index.html#platform-specific-tutorials

**Comment by xiaoxiang781216** [comment]
@s-hadinger should we move berry to binding section instead?
https://docs.lvgl.io/master/get-started/bindings/index.html

**Comment by s-hadinger** [comment]
Hi, sorry. Yes. Sure

**Pull request merged**